care/facility/tests/test_patient_sample_api.py

```diff
@@ -0,0 +1,130 @@
+import datetime
+
+from django.utils.timezone import make_aware
+from freezegun import freeze_time
+from rest_framework import status
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5356032)